src/store/cpak.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::errors::Error;
-use jsonwebkey::JsonWebKey;
+use jsonwebtoken::{self as jwt, jwk};
```

Comment:
Do we use any JWT-related functionality?  If not, this could be streamlined as:

```suggestion
use jsonwebtoken::jwk;
```